config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :mechanics, only: :show
+  post "/mechanics/:id", to: "mechanics#edit"
```

Comment:
a more RESTful option for US 2 (see controller comments) could be:
`post "/mechanics/:mechanic_id/rides", to: "ride_mechanics#create"`